02.calendar/calendar.js

```diff
@@ -0,0 +1,45 @@
+import { program } from "commander";
+
+program
+  .option("-m, --month <n>", "1~12を入力", checkInput)
+  .option("-y, --year <n>", "1970~2100を入力", checkInput);
+program.parse();
+const options = program.opts();
+
+const today = new Date();
+let year;
+let month;
+
+if (options.year !== undefined) {
+  if (options.year >= 1970 && options.year <= 2100) year = options.year;
+} else year = today.getFullYear();
```

Comment:
ちょっと読みづらいので、
```suggestion
if (options.year !== undefined) {
  if (options.year >= 1970 && options.year <= 2100) {
    year = options.year;
  }
} else {
  year = today.getFullYear();
}
```
のようにしてもらうほうがよいかと思いました。